### PR TITLE
New python task

### PR DESCRIPTION
**Pull request by talarian1**
- [x] All [tests](https://github.com/jfrog/artifactory-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: tasks/ArtifactoryPython/pythonBuild.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            performPipInstall(cliPath);
+            break;
+    }
+}
+
+function performPipInstall(cliPath:string) {
+    let inputWorkingDirectory = tl.getInput('workingDirectory', false);
+    let defaultWorkDir = tl.getVariable("System.DefaultWorkingDirectory") || process.cwd();
+    let sourcePath = utils.determineCliWorkDir(defaultWorkDir, inputWorkingDirectory);
+    let configuredServerId = performPipConfig(cliPath, sourcePath);
```
Comment (eyalbe4) [inline tasks/ArtifactoryPython/pythonBuild.ts:26]: If we move this line into the below ```executeCliCommand``` function, we won't need to pass the ```configuredServerId``` as an argument to this function.

File: tasks/ArtifactoryPython/task.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
+{
+    "id": "9605c9d9-b8a7-45b1-9887-89c330f138c9",
+    "name": "ArtifactoryPython",
+    "friendlyName": "Artifactory Python",
+    "description": "Build and publish Python packages from and to Artifactory while allowing to collect build-info. The collected build-info can be later published to Artifactory by the \"Artifactory Publish Build Info\" task.",
```
Comment (eyalbe4) [inline tasks/ArtifactoryPython/task.json:5]: I suggest rephrasing the message as follows:

Run 'pip install' while resolving the python dependencies from Artifactory. This task also allows collecting build-info. The collected build-info can be later published to Artifactory by the \"Artifactory Publish Build Info\" task.",

File: tests/resources/python/install.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+let inputs = {
+    buildName: 'Python Test',
+    buildNumber: '17',
+    command: 'install',
+    arguments: '-r requirements.txt --module=jfrog-python-example',
+    workingDirectory: path.join(testUtils.getLocalTestDir(TEST_NAME)),
+    targetResolveRepo: testUtils.getRepoKeys().pythonVirtualRepo,
+    collectBuildInfo: true,
+    noPipCache: true
+
```
Comment (eyalbe4) [inline tests/resources/python/install.js:14]: Redundant new-line.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

**Review by yahavi — CHANGES_REQUESTED** [review]
Awesome work @talarian1!
Please consider my suggestions. Also, I'd suggest to change all ArtifactoryPython to ArtifactoryPip.
Please also run `npm run format`.

**Pull request merged**